BackofficeBundle/Form/DataTransformer/ChoicesOptionToArrayTransformer.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class ChoicesOptionToArrayTransformer
+ */
+class ChoicesOptionToArrayTransformer implements DataTransformerInterface
+{
+    protected $suppressSpecialCharacterClass;
+
+    /**
+     * @param SuppressSpecialCharacterTransformer $suppressSpecialCharacterClass
+     */
+    public function __construct(SuppressSpecialCharacterTransformer $suppressSpecialCharacterClass)
+    {
+        $this->suppressSpecialCharacterClass = $suppressSpecialCharacterClass;
+    }
+
+    /**
+     * @param array $arrayChoices
+     * @return string
+     */
+    public function transform($arrayChoices)
+    {
+        if (null === $arrayChoices) {
+            return "";
+        }
+
+        return implode(',', $arrayChoices);
+    }
+
+    /**
+     * @param string $choices
+     * @return array choix1,choix2,choix3
```

Comment:
just put the @return array tag

If you want to be more clear, either the test should do it or you can write a little comment about it